win32ss/gdi/eng/gradient.c

```diff
@@ -396,6 +398,7 @@ IntEngGradientFillTriangle(
               ENDLINE(v1, v2, 1);
 
               GOLINE(v2, v3, 2);
+              FILLLINE(0, 2);
```

Comment:
Only to double-check, is this change still needed after the other ones?

Reply:
Yes. Without this line there are still some unfilled areas of each circle. See Below:

<img width="1042" height="856" alt="gradientfill-no-extra-FILLINE" src="https://github.com/user-attachments/assets/2988f997-2b8b-47cc-82bc-09382fadaab3" />

Reply:
Ok.